src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class AdminClientLogging extends Logger {
```

Comment:
@jbudz is right, we would need to have the export be a factory with the server object. I added [unit testing](https://github.com/elastic/kibana/pull/9132/commits/98553213a0742d5aeb2a41953a4d8f12d5e11a27) of create_{client,admin}_cluster.js including the logger with the existing implementation.